### PR TITLE
Convert epel role to install_epel_online

### LINKED ISSUE #46: Convert epel role to install_epel_online

### PR DESCRIPTION
**Pull request by Dostonbek1**
Closes #46
- [ ] `cd install/; ./install.sh`
- [ ] `cd install/; ./install.sh -e install_offline=true`

### CODE REVIEW

File: install/roles/install_epel_online/tasks/main.yml [hunk 1]
```diff
@@ -9,15 +9,6 @@
...
-- name: Test for rpm command
-  shell: command -v rpm
-  register: have_rpm_raw
-  ignore_errors: yes
-
-- name: Set have_rpm
-  set_fact:
-    have_rpm: "{{ have_rpm_raw.rc == 0 }}"
-
 - name: Check for EPEL rpm (RHEL/Centos)
```
Comment (myersCody) [inline install/roles/install_epel_online/tasks/main.yml]: I think you should rename this to "check if epel is installed with yum"

File: install/roles/install_epel_online/tasks/main.yml [hunk 2]
```diff
@@ -9,16 +9,7 @@
   set_fact:
     have_yum: "{{ have_yum_raw.rc == 0 }}"
 
-- name: Test for rpm command
-  shell: command -v rpm
-  register: have_rpm_raw
-  ignore_errors: yes
-
-- name: Set have_rpm
```
Comment (myersCody) [inline install/roles/install_epel_online/tasks/main.yml]: Your current code still uses this variable even though you deleted it. Therefore, this task will now never run. 
```
- name: Check if EPEL is installed with yum (RHEL/Centos)
  yum:
    name: epel-release
    state: installed
  register: have_epel_rpm_raw
  become: true
  ignore_errors: yes
  when:
    - have_rpm|bool
```

File: install/roles/install_epel_online/tasks/main.yml [hunk 3]
```diff
@@ -40,35 +31,6 @@
   when:
     - have_epel_rpm is not defined
```
Comment (myersCody) [inline install/roles/install_epel_online/tasks/main.yml]: Why do these variables reference rpm if we now use yum?

File: install/roles/install_epel_online/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  set_fact:
+    have_yum: "{{ have_yum_raw.rc == 0 }}"
+
+- name: Check if EPEL is installed with yum (RHEL/Centos)
+  yum:
+    name: epel-release
+    state: installed
+  register: have_epel_raw
+  become: true
+  ignore_errors: yes
```
Comment (myersCody) [inline install/roles/install_epel_online/tasks/main.yml]: I still think that you need a when condition here, this will fail if they do not have yum.

Comment (Dostonbek1) [reply install/roles/install_epel_online/tasks/main.yml]: I added when condition back in and tested both online and offline. They both worked fine without any fails.

File: install/qpc_playbook.yml [hunk 5]
```diff
@@ -19,7 +19,7 @@
       ansible_log_level: 0
   roles:
     - prep
-    - epel
+    - install_epel_online
```
Comment (myersCody) [inline install/qpc_playbook.yml]: This is running on both online and offline when I test it.

**Comment by myersCody** [comment]
The offline install does not check epel_release is installed.
```
TASK [qpc_cli : Install QPC from local path] ***********************************************************************************************************************************************************************************************************************************
 [WARNING]: Consider using the yum, dnf or zypper module rather than running 'rpm'.  If you need to use command because yum, dnf or zypper is insufficient you can add 'warn: false' to this command task or set 'command_warnings=False' in ansible.cfg to get rid of this
message.

fatal: [localhost]: FAILED! => {"changed": true, "cmd": "rpm -Uvh --force \"/quipucords_installer/install/packages/qpc.el7.noarch.rpm\"", "delta": "0:00:00.212178", "end": "2019-07-09 08:43:34.758360", "msg": "non-zero return code", "rc": 1, "start": "2019-07-09 08:43:34.546182", "stderr": "warning: /quipucords_installer/install/packages/qpc.el7.noarch.rpm: Header V3 RSA/SHA1 Signature, key ID d7c8b724: NOKEY\nerror: Failed dependencies:\n\tepel-release is needed by qpc-0.9.0-1.git.59.571eacf.el7.noarch", "stderr_lines": ["warning: /quipucords_installer/install/packages/qpc.el7.noarch.rpm: Header V3 RSA/SHA1 Signature, key ID d7c8b724: NOKEY", "error: Failed dependencies:", "\tepel-release is needed by qpc-0.9.0-1.git.59.571eacf.el7.noarch"], "stdout": "", "stdout_lines": []}
```
I left steps to reproduce in the description.

**Pull request merged**